documentation/book/con-strategies-for-upgrading-clients.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrading-kafka-versions.adoc
+
+[id='con-strategies-for-upgrading-clients-{context}']
+
+= Strategies for upgrading clients
```

Comment:
While reading this end-to-end, I felt that this section interrupted the flow of information and my awareness of where I was in the overall upgrade procedure. I would consider moving it to the end of this assembly, or into its own section.

Reply:
That's an interesting observation. I have moved to the end of the assembly. When we do the restructure, this could be something to add to an appendix out of the main flow entirely.